### ISSUE TITLE
Generate dataset label

### ISSUE THREAD

**Issue by Therock90421** [ying09/TextFuseNet.pytorch#79]
Hi, thank you for your excellent work!
When i wanna train TextFuseNet on my own dataset, it puzzles me that whether need I should put segmentation label into Ground-Truth json. Should I only use word and character level labels to generate gt-label? Thanks again.

**Comment by Real-YeJ** [comment]
@Therock90421  put your detection results of characters as pseudo labels, and refer to the json format of COCO.
